src/products/utils/handlers.ts

```diff
@@ -120,6 +124,35 @@ export function createAttributeMultiChangeHandler(
   };
 }
 
+export function createAttributeFileChangeHandler(
+  changeAttributeData: FormsetChange<string[]>,
+  attributesWithNewFileValue: FormsetData<FormsetData<null, File>>,
+  addAttributeNewFileValue: (data: FormsetAtomicData<null, File>) => void,
+  removeAttributeNewFileValue: (id: string) => void,
+  triggerChange: () => void
+): FormsetChange<File> {
+  return (attributeId: string, value: File) => {
+    triggerChange();
+    if (value) {
```

Comment:
personally I think that if / else inside of if / else makes the logic more compliated, maybe we could 
```
if(value) { 
   // do stuff
  return;
}

other stuff